templates/deployment-postgres.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
```

Comment:
See below, We should use maas's postgre chart for infrastructure so we are not duplicating.